site/docs/pages/wallet/wallet.mdx

```diff
@@ -317,49 +319,23 @@ import { color } from '@coinbase/onchainkit/theme';
   </Wallet>
 </WalletComponents>
 
-### Implementing the Wallet Modal for wallet connection aggregation
+## Using Wallet Modal
+
+<img alt="Wallet Modal"
+  src="https://onchainkit.xyz/assets/wallet-modal.gif"
+  height="364"/>
 
-OnchainKit provides a built-in wallet modal that offers users multiple connection options. The modal automatically appears when users click the `ConnectWallet` component and provides three distinct connection paths:
+Wallet modal offers users multiple wallet connection options. The modal automatically appears when users click the `ConnectWallet` component and provides three distinct connection paths:
 
 1. Quickly create a new smart wallet for new users
 2. Coinbase Wallet connection (supporting both smart wallets and EOA)
-3. Other wallets via WalletConnect
+3. Other wallets via [WalletConnect](https://docs.walletconnect.org/)
 
-#### Setup Steps
+### Setup Steps
 
-1. Configure the wallet connectors in your Wagmi config:
-
-```tsx [wagmiConfig.ts]
-const wagmiConfig = createConfig({
-  chains: [base, baseSepolia],
-  connectors: [
-    // Smart wallet support
-    coinbaseWallet({
-      appName: 'Your App',
-      preference: 'smartWalletOnly',
-    }),
-    // EOA wallet support
-    coinbaseWallet({
-      appName: 'Your App',
-      preference: 'eoaOnly',
-    }),
-    // Other wallets via WalletConnect
-    walletConnect({ // [!code focus]
-      projectId: process.env.WALLET_CONNECT_PROJECT_ID, // [!code focus]  
-      showQrModal: true, // [!code focus]
-      metadata: { // [!code focus]
-        name: 'Your App', // [!code focus]
-        description: 'Your app description', // [!code focus]
-        url: 'https://your-app.com', // [!code focus]
-        icons: [], // [!code focus]
-      },
-    }),
-  ],
-  // ... other config
-});
-```
+::::steps
 
-2. Configure the `OnchainKitProvider` with modal settings:
+## Configure the `OnchainKitProvider` with modal settings:
```

Comment:
the parent here is a three-hash (`### Setup Steps`) so i'd make this a four-hash